build.sbt

```diff
@@ -52,7 +52,7 @@ lazy val coverageSettings: Seq[Setting[_]] = {
 val compile = Seq(
   ws,
   "uk.gov.hmrc"       %% "bootstrap-backend-play-28" % bootstrapPlayVersion,
-  "com.typesafe.play" %% "play-json-joda"            % "2.6.14"
+  "com.typesafe.play" %% "play-json-joda"            % "2.10.0-RC7"
 )
 
 def test(scope: String = "test,it"): Seq[ModuleID] = Seq(
```

Comment:
Are there any higher versions here we can use? Don't worry about it if they cause errors but it would be nice to bump some of them up if we can (jsoup goes up to 1.15.3 for instance)